src/components/App/ExpressRecorder.tsx

```diff
@@ -337,14 +337,14 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
         );
     };
 
-    stopStreams = () => {
+    stopStreams = (screenOn?: boolean) => {
         const { stream, screenStream } = this.state;
         if (stream) {
             stream.getTracks().forEach(function(track) {
                 track.stop();
             });
         }
-        if (screenStream) {
+        if (screenStream && !screenOn) {
             screenStream.getTracks().forEach(function(track) {
                 track.stop();
```

Comment:
I don't understand variable names here.
please change those to something with a concrete name and comment on the meaning of each condition.
(change `stream`, `screenStream`, `screenOn`, `stopStreams`)

Reply:
if it's too hard to change variable names, then comment with some explanation about the flow here.

Reply:
done